models/staging/graph/stg_exposure_relationships.sql

```diff
@@ -1,17 +1,14 @@
-with relationships as (
-
-    {{
-        dbt_project_evaluator.get_relationships("exposures")
-    }}
+with 
 
+_base_exposure_relationships as (
```

Comment:
Why the `_` prefix?

Reply:
Duckdb did not like the same named CTE and table for some reason!